server/adaptors/integrations/__data__/repository/aws_cloudtrail/aws_cloudtrail-1.0.0.json

```diff
@@ -60,46 +66,67 @@
   ],
   "assets": [
     {
-      "name": "aws_cloudtrail",
-      "version": "1.0.0",
-      "extension": "ndjson",
-      "type": "savedObjectBundle",
-      "workflows": ["dashboards"]
-    },
-    {
-      "name": "aws_cloudtrail-flint",
+      "name": "create_table_cloud-trail",
       "version": "1.0.0",
-      "extension": "ndjson",
-      "type": "savedObjectBundle",
+      "extension": "sql",
+      "type": "query",
       "workflows": ["dashboards-flint"]
     },
     {
-      "name": "create_skipping_index",
+      "name": "create_table_cloud-trail-records",
       "version": "1.0.0",
       "extension": "sql",
       "type": "query",
-      "workflows": ["queries"]
+      "workflows": ["dashboards-flint-records"]
     },
     {
-      "name": "example_queries",
+      "name": "create_skipping_index",
       "version": "1.0.0",
-      "extension": "ndjson",
-      "type": "savedObjectBundle",
+      "extension": "sql",
+      "type": "query",
       "workflows": ["queries"]
     },
     {
-      "name": "create_table_cloud-trail",
+      "name": "create_mv_cloud-trail",
       "version": "1.0.0",
       "extension": "sql",
       "type": "query",
       "workflows": ["dashboards-flint"]
     },
     {
-      "name": "create_mv_cloud-trail",
+      "name": "create_mv_cloud-trail-records",
```

Comment:
Just for my knowledge, what is the record referencing to?

Reply:
extended format for multiple records shown [here](https://aws.amazon.com/blogs/big-data/aws-cloudtrail-and-amazon-athena-dive-deep-to-analyze-security-compliance-and-operational-activity/)

Reply:
Do I understand right that we only have acceleration for single-record, and multi-record has queries without acceleration? LGTM technically, but I wonder how common multi-record is compared to single-record/what the impact of that is.